Makefile

```diff
@@ -141,6 +141,11 @@ build-docker-wasm:
 
 .PHONY: build-docker-wasm
 
+build-wasm-light-clients:
+	cd modules/light-clients/08-wasm-light-clients && cargo wasm
```

Comment:
why don't we have a build script that uses cosmwasm optimizer? I think this is the only one we need since we want reproducible builds

Reply:
I was planning to add that when this is together with e2e integration, but it does take quite a bit longer to run, and it is not strictly necessary for CI right now.